game-core/src/main/java/games/strategy/triplea/ui/UserActionPanel.java

```diff
@@ -121,11 +121,11 @@ public void actionPerformed(final ActionEvent event) {
       final JPanel userChoicePanel = new JPanel();
       userChoicePanel.setBorder(BorderFactory.createEmptyBorder(8, 8, 8, 8));
       userChoicePanel.setLayout(new GridBagLayout());
-      int row = 0;
 
       final JScrollPane choiceScroll = new JScrollPane(getUserActionButtonPanel(userChoiceDialog));
       choiceScroll.setBorder(BorderFactory.createEtchedBorder());
       choiceScroll.setPreferredSize(getUserActionScrollPanePreferredSize(choiceScroll));
+      int row = 0;
```

Comment:
In this case, I think `row` being declared before the first component "paragraph" is the more understandable reading.

Reply:
8c16a09